SAT.js

```diff
@@ -498,11 +498,11 @@
     var w = this['w'];
     var h = this['h'];
     return new Polygon(new Vector(pos['x'], pos['y']), [
-     new Vector(), new Vector(w, 0), 
-     new Vector(w,h), new Vector(0,h)
+     new Vector(pos['x'], pos['y']), new Vector(pos['x'] + w, pos['y']),
```

Comment:
This isn't correct. The `points` passed into a `Polygon` are all _relative_ to the `pos` vector, they are not absolute positions.

Your change offsets the points of the polygon by an additional `pos` amount, which will put them in an incorrect position - it will only work if `pos` is `Vector(0,0)`.